test/map.test.ts

```diff
@@ -68,6 +68,7 @@ describe('map', function() {
 
   it('should ignore enumerable properties on arrays', function() {
     var array = ['a', 'b', 'c'];
+    // @ts-ignore
```

Comment:
I actually have no idea what this test is trying to accomplish.  I've never seen dot notation syntax on an array like this.